sdk/sdk-api/src/main/java/mil/emp3/api/utils/kml/KMLExportThread.java

```diff
@@ -1267,7 +1267,7 @@ private void exportEmpObjectToKML(final IContainer container, XmlSerializer xmlS
         }
     }
 
-    private String export(IMap map, XmlSerializer xmlSerializer)
+    String export(IMap map, XmlSerializer xmlSerializer)
```

Comment:
same question here, why has everything changed from private in the methods to essentially public